### ISSUE TITLE
Windows 8.x touch compability issues

### ISSUE THREAD

**Issue by andersevenrud** [andersevenrud/OS.js-v2#111]
A user has been reporting (by mail) that the nightly is not working as it should on Chrome (Windows).

Strangely enough it does not happen on the demo-site and I am unable to reproduce this.

Reports of not being able to move the window or close it with the button(s).

No errors are logged which leads me to believe that the input events are simply not registered.

I know the function for this has some compability code... maybe it's not working correctly.

**Video:** http://webm.host/aae04/

**Comment by andersevenrud** [comment]
I narrowed down to the touch-code compatibility. When thinking about it it's actually not weird he's getting these at all. I "updated" the compability code in df0490c8d996957fd170de485119dbdb1404a265 and changed it ever so slightly. Turns out this triggers a bug in certain Chrome versions. I have now changed this in 387edf4ef915b293f1f8b44b39f1cf196057b458. Waiting for the fine gentleman to ping me back with a report.
